examples/lib/stories/rendering/isometric_tile_map.dart

```diff
@@ -7,45 +7,29 @@ import 'package:flame/input.dart';
 import 'package:flame/sprite.dart';
 import 'package:flutter/material.dart' hide Image;
 
-const x = 500.0;
-const y = 500.0;
-final topLeft = Vector2(x, y);
+class IsometricTileMapExample extends FlameGame with MouseMovementDetector {
```

Comment:
I was going to add a tiled example, that is why I created tile_maps :P 
but back how we were doing it then I was unable to add a flame_x package as dep of this package

Reply:
The `flame_tiled` example should be in the `example` folder of the package now? :)

Reply:
I think each package has its own `example` folder with a simple example of that package only, but I wanted to include more examples for all packages and their interactions on the examples app

Reply:
I second this idea.

The individual examples are not so accessible, would be nice that they were bundled inside our Dashbook too.

Reply:
Sounds good! But we can bring back this section when there are more examples then one then?